docs/user_guide/bank/community_product_drive_participants.md

```diff
@@ -0,0 +1,38 @@
+# Product Drive Participants
+
+If you conduct product drives, the product drive participants list lets you manage the participant's contact information and see their donation history.
+
+To manage product drive participants, click on " Product Drive Participants" under the "Community" section.
+
+![Product Drive Participants](images/community/product_drive_participants/product_drive_page.jpg)
+
+This page lists some basic details from all previously entered product drive participants -- business, contact, phone email, and total items, letting you drill down to more information, or to edit each participant's information.
+
+## Adding participants
+
+To add a new participant, click on the "+ New Product Drive Participant" button, add their details including the business name, contact name, phone, email, address and comments.
+
+![Add Participant](images/community/product_drive_participants/add_participant.jpg)
+
+After saving the participant's details there will be a new row in the Product Drive Participants page.
+
+You can also add new participants "on the fly" as you enter the donations, through the new [donation screen](essentials_donations.md).
```

Comment:
Small tweak -- please include new in the link, and capitalize New and Donation.  (Reason:  It's the "New Donation" page,  not a "donation" page that is new, which is implied by having the new outside  of it.)

This is also where I wanted that "TODO" notation,  wasn't it?